packages/storybook/.storybook/preview.ts

```diff
@@ -1,19 +1,18 @@
-import { Preview, ReactRenderer } from '@storybook/react';
-import { withThemeByClassName } from '@storybook/addon-themes';
+import { Preview, ReactRenderer } from "@storybook/react";
+import { withThemeByClassName } from "@storybook/addon-themes";
 
-import '@dailydotdev/shared/src/styles/globals.css';
+import "@dailydotdev/shared/src/styles/globals.css";
```

Comment:
I think our linting is not set right on this mono, it should prefer single quotes actually. following the rest of the monorepo.

Reply:
Found the issue - it was bad `package.json` file in the storybook directory. Should be fixed now 👍